QueAns/tests.py

```diff
@@ -0,0 +1,135 @@
+from datetime import timezone, timedelta
+from unittest import TestCase
+
+from QueAns.views import upvote, downvote, satisfied, disatisfied
+from .models import Meetings,Question
+from django.contrib.auth.models import User
+
+
+def create_Meeting(meeting_text,venue,moderator,start_time,end_time):
+    return Meetings.objects.create(meeting_text,venue,moderator,start_time,end_time)
+
+def create_Question(question_text,meeting_id,asker,anon_status):
+    return Question.objects.create(question_text,meeting_id,asker,anon_status)
+
+def create_User(username,password):
+    user = User.objects.create_user(username,password)
+    user.is_staff = True
+    return user
+
+u1 = create_User('dummy','dummy')
+u2 = create_User('dummy2','dummy2')
+m1 = create_Meeting('dummy','homer','dummy',timezone.now(),timezone.now()+timedelta(days=1))
+m2 = create_Meeting('dummy2','homer','dummy',timezone.now()-timedelta(days=1),timezone.now())
+q1 = create_Question('dummyQ',m1.id,'dummy','no')
+q2 = create_Question('dummyQ',m2.id,'dummy','yes')
+
+class upvote_test(TestCase):
+    def new_vote(self):
```

Comment:
When you run tests, are these actually getting run? Is there is a pattern in this your test functions should always be in django/unit test